installer/x86_64/install.sh

```diff
@@ -92,6 +92,39 @@ VAR_LOG_SIZE=4096
 
 [ -r platforms/$onie_platform ] && . platforms/$onie_platform
 
+# Verify image platform is inside devices list
+# Temporarily set TIMEOUT as 10 seconds
+if [ "$install_env" = "onie" ] && [ -d devices ]; then
```

Comment:
>onie [](http://example.com/codeflow?start=23&length=4)

Is it possible to run on all the envrionment besides ONIE?

Reply:
It could be an option among sonic/onie/build

Reply:
We have sonic_installer to block wrong platform already. It might be tedious if user input --skip-platform-check and be prompted here the second time to confirm install.

Reply:
Reasonable. Could you change sonic-utilities code with same logic, checking platform_asic instead of previous image asic.

Reply:
This comment could be closed.